### PR TITLE
Milestone: Support Responsive Web Design 💻📱🤩

### PR DESCRIPTION
**Pull request by ahmedeid6842**
- Added Media queries to Each section to achieve RWD
- Updated Header section if min-width > 768
  -  Added navigation list
  - change texts style to fix the new width 
- Updated Works section in case min-width > 768
  - Updated the width for each work card to fit the width
  - Each work card is split between  work image and work details
  - Align the work image on the card's right and the work details on the left then mirror it for the next card 
- Updated About section in case min-width>768
  -  Moved languages,... Drop down next to About myself 
- Updated Contact form section in case min-width > 768
  - Align the inputs field to the form center
- Updated a README documentation 
- Updated `index` and `style.css` files with some styling 

### Finally 😅😅

jim-carrey-finally

### CODE REVIEW

File: styles/main.css [hunk 1]
```diff
@@ -253,6 +315,109 @@ header {
...
+  }
+
+  .card:nth-child(2n) {
+    flex-direction: row-reverse;
+  }
+
+  .card__imgbox {
+    height: 100%;
+    width: 50%;
+  }
```
Comment (zieeco) [inline styles/main.css:347]: - Hey, let's try to improve our design by adjusting the styles on the image as it goes out of the card.

*Your card*
Screenshot from 2023-05-04 18-44-12

*your design*
image

File: index.html [hunk 2]
```diff
@@ -16,46 +16,62 @@
...
+                    <li>Contact</li>
+                </a>
+            </ul>
+
+            <div class="nav-menu">
                 <img src="./images/menu.svg" alt="menu-icon" class="menu">
             </div>
         </nav>
 
     </header>
```
Comment (zieeco) [inline index.html:38]: - Your styles are not reflecting on GitHub, kindly update the link to 
`<link rel="stylesheet" href="./styles/main.css">` to fix this issue.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @ahmedeid6842,



Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._




## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -136,57 +153,62 @@ <h4 class="card__canopy-title">CANOPY</h4>
...
+            </div>
         </div>
         <ul class="drop__down">
             <li class="language__list">
                 <span>Languages</span>
                 <i class='fas fa-angle-down'></i>
             </li>
-            <li class="language_list_technologies">
-                <img src="./images/JavaScript.svg" alt="">
-                <span>JavaScript</span>
```
Comment (Efiamotu-1) [inline index.html:180]: - [ ] your desktop design for the technologies section in your about me is largely inconsistent with the figma template as shown below please kindly work on it as it is very important to always achieve the given figma design to the latter :+1: 
- figma design
in-f
- your design
in-s

File: index.html [hunk 4]
```diff
@@ -136,57 +153,62 @@ <h4 class="card__canopy-title">CANOPY</h4>
...
+                    class="btn ">Get me resume</a>
+            </div>
         </div>
         <ul class="drop__down">
             <li class="language__list">
                 <span>Languages</span>
                 <i class='fas fa-angle-down'></i>
             </li>
-            <li class="language_list_technologies">
-                <img src="./images/JavaScript.svg" alt="">
```
Comment (Efiamotu-1) [inline index.html:179]: - [ ] your mobile design for the technologies section is not consistent with the provided figma template please make sure to fix this as well :+1: 
- figma design 
in-mf
- your design
in-m

File: index.html [hunk 5]
```diff
@@ -16,46 +16,62 @@
...
+                </li>
+                <li>
+                    <a href="https://github.com/ahmedeid6842" rel="noopener noreferrer" target="_blank">
+                        <img src="images/github.svg" alt="github-logo">
+                    </a>
+                </li>
+            </ul>
+        </div>
     </section>
-    <section class="work">
```
Comment (Efiamotu-1) [inline index.html:53]: - [ ] There is little spacing between the header and the works section in your desing however this is not the same as in the case in the figma template please try to design your web page to match the design of the figma template :+1: 
- figma design
fig-d
- your design
fig-s

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @ahmedeid6842 >, 



Great work on making the changes requested by a previous reviewer 👏 Your Code is Awesome! 💪
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Especially if that's is your first project 1️⃣. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better job** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- Good commit history :clap: 
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html
Comment (Efiamotu-1) [inline index.html:1]: [OPTIONAL]
- I will suggest that you use this particular css stlying for the margin `margin: 2.5rem 0;` in your line `97` of `main.css` to achieve the design shown below becasue currently there is margin to the left and right of your header :+1:  :grin: 
ach

**Review by Efiamotu-1 — APPROVED** [review]
Hi @ahmedeid6842 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# STATUS: APPROVED 🕺🥳 

giphy


## Highlights :dart: 
* Correct GitHub flow :heavy_check_mark: 
* Good commit messages :heavy_check_mark: 
* All linters passing :heavy_check_mark: 
* Good commit history :100: 
* Good PR summary :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, you can now Merge since it is Approved ✅, However if there are any conflicts please resolve them before merging.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ahmedeid6842** [comment]
7fb4d55fa09fc4f5426e6d5934d1d02f

**Comment by bellom** [comment]
Hello @ahmedeid6842 👋,

_This is an additional review as a result of the second opinion request._

Sorry for the late response, this is due to the hour you asked for the SO.

I'm happy 🙂  to see your project has been approved, and thanks for bringing this to us we will reach out to the CR.

Thank you.

Keep rocking 🚀!

**Pull request merged**